burgerqueen/src/components/OrderReadyItem.jsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { Icon } from '@iconify/react';
+
+const OrderReadyItem = (props) => {
+  const { order, scrl } = props;
+  return (
+    <div className="container-info" ref={scrl}>
+      <h3>{order.client}</h3>
+      <div className="status-closeBtn">
+        <p>{order.status}</p>
+        <button type="button" className="btn">
+          <Icon icon="carbon:close-filled" color="#f2884b" height="40" />
+        </button>
+      </div>
+    </div>
+  );
+};
+
+export default OrderReadyItem;
```

Comment:
scrl no es un nombre muy descriptivo
```suggestion
import { Icon } from '@iconify/react';

const OrderReadyItem = ({order}) => {
  const { client, status } = order
  return (
    <div className="container-info" ref={scrl}>
      <h3>{client}</h3>
      <div className="status-closeBtn">
        <p>{status}</p>
        <button type="button" className="btn">
          <Icon icon="carbon:close-filled" color="#f2884b" height="40" />
        </button>
      </div>
    </div>
  );
};

export default OrderReadyItem;
```